lib/auth/keystore/aws_kms.go

```diff
@@ -141,6 +151,16 @@ func (a *awsKMSKeystore) generateRSA(ctx context.Context, _ ...rsaKeyOption) ([]
 	return keyID, signer, nil
 }
 
+func awsAlgorithm(alg cryptosuites.Algorithm) (string, error) {
+	switch alg {
+	case cryptosuites.RSA2048:
+		return kms.KeySpecRsa2048, nil
+	case cryptosuites.ECDSAP256:
+		return kms.KeySpecEccNistP256, nil
+	}
+	return "", trace.BadParameter("unsupported algorithm: %v", alg)
```

Comment:
Currently it will error here. I'm planning to make some improvements later on to warn or error if the suite isn't valid based on whether you've configured an HSM/KSM or are running FIPS